epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -53,69 +55,82 @@
-				document also establishes formal requirements for contents to be certified as accessible. These
-				requirements provide EPUB creators with a clear set of guidelines for evaluating their content with
-				respect to quality certification and enable it.</p>
-
+			
+			<p>The EPUB Accessibility defines requirements for accessibility metadata. In this way consumers can examine the accessibility characteristics of the content and make well-informed decisions on whether an EPUB publication is appropriate for each need before buying, borrowing or downloading it.
+				This metadata, attached to the digital file and distributed along the supply chain, ensures that persons with disabilities can be informed about the publication accessibility features.
+				In order to make sure that information gets to the end user, it is critical that all players in the value chain do their part in managing metadata and displaying it to the end user (aggregators, ecommerce, reading solutions).</p>
+
+			<p>EPUB Accessibility also defines the official accessibility criteria that EPUBs must adhere to claim conformance to the standard.
+				These criteria offer EPUB creators a precise set of instructions to assess their content in terms of quality assurance.</p>
+			
 			<p>The EPUB Accessibility does not target a single version of EPUB (most commercially available publications
 				are in EPUB 3 or EPUB 2 format). It is designed to be applicable to all EPUB publications conforming to
 				any version or profile, including future versions of the standard.</p>
+				
+			<p>The EPUB allows to create both reflowable and Fixed Layout publications.</p>
+			
+			<p>In a reflowable EPUB the content adapts to the user's viewport, depending also on the user's chosen graphical characteristics, such as font size, line spacing, text color, etc.; while in Fixed Layout EPUBs each element is positioned absolutely on the page (as it is usually an exact replica of the paper version) and the user has not the ability to change the text formatting.</p>
+
+			<p>EPUB Fixed Layout are mainly used for publications in which the meaning is layout-driven such as comics, illustrated children’s books and art books.</p>
+
+			<p>Although it is technically possible to produce Fixed Layout publications with a good level of accessibility, it should be kept in mind that some accessibility requirements cannot be met, precisely because of the limitation of the format, such as the ability to change the graphical formatting of the text according to the user's needs.</p>
+		
+			<section id="relationship-to-wcag">
+				<h2>Relationship of EPUB Acessibility to WCAG</h2>
+				
+				<p>EPUB Accessibility specification defines the requirements necessary to make an EPUB publication fully accessible including to the ones defined by the Web Content Accessibility Guidelines (WCAG) and adding the ones related with the specific characteristic of ebooks and digital publications.</p>
```

Comment:
```suggestion
				<p>EPUB Accessibility specification defines requirements necessary to make an EPUB publication fully accessible; these are on top of the requirements defined by the Web Content Accessibility Guidelines (WCAG) and relate to the specificificities of ebooks and digital publications.</p>
```

(I hope I got the intended meaning right.)